authentication/src/main/java/io/camunda/authentication/config/AudienceValidator.java

```diff
@@ -39,7 +41,7 @@ final class AudienceValidator implements OAuth2TokenValidator<Jwt> {
 
   @Override
   public OAuth2TokenValidatorResult validate(final Jwt token) {
-    final var tokenAudiences = token.getAudience();
+    final var tokenAudiences = Objects.requireNonNullElse(token.getAudience(), List.<String>of());
```

Comment:
🔧 NABD but in this instance I think the intent is clearer if you use `Collections.emptyList()`